FindNearby/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "salesforce-app",
+  "private": true,
+  "version": "1.0.0",
+  "description": "Salesforce App",
+  "scripts": {
+    "lint": "eslint **/{aura,lwc}/**",
+    "test": "npm run test:unit",
+    "test:unit": "sfdx-lwc-jest",
+    "test:unit:watch": "sfdx-lwc-jest --watch",
+    "test:unit:debug": "sfdx-lwc-jest --debug",
+    "test:unit:coverage": "sfdx-lwc-jest --coverage",
+    "prettier": "prettier --write \"**/*.{cls,cmp,component,css,html,js,json,md,page,trigger,xml,yaml,yml}\"",
+    "prettier:verify": "prettier --list-different \"**/*.{cls,cmp,component,css,html,js,json,md,page,trigger,xml,yaml,yml}\"",
+    "postinstall": "husky install",
```

Comment:
`husky install` can only install git hooks at the top of a git project. If you run `npm install` in this folder, you get errors at the end of the install:

```
> salesforce-app@1.0.0 postinstall
> husky install

.git can't be found (see https://git.io/Jc3F9)
npm ERR! code 1
npm ERR! path /<Path to repo>/LWC-Mobile-Samples/FindNearby
npm ERR! command failed
npm ERR! command sh -c -- husky install

npm ERR! A complete log of this run can be found in:
npm ERR!     /<Home Dir>/.npm/_logs/2023-01-14T01_05_30_225Z-debug-0.log
```

I don't see any Husky-related hooks to configure anyway, so this should probably just be removed.